dnssec.go

```diff
@@ -556,9 +556,9 @@ func (k *DNSKEY) publicKeyRSA() *rsa.PublicKey {
 	pubkey := new(rsa.PublicKey)
 
 	var expo uint64
-	for i := 0; i < int(explen); i++ {
+	for _, v := range keybuf[keyoff:modoff] {
```

Comment:
found that `explen` give some extra documentation here, add comment that this loops is executed `explen` times?

Reply:
@miekg Can do. I’ll document it tomorrow when I’m back at my computer.

Reply:
@miekg How is that comment?